forge/routes/api/project.js

```diff
@@ -688,6 +691,9 @@ module.exports = async function (app) {
      * @name /api/v1/project/:id/export
      * @memberof forge.routes.api.project
      */
+    /*
+       @Steve-Mcl @hardillb "will need updating to check if there is a credentialSecret in the ProjectSettings tables first and only look in the StorageSettings entry if it's not found"
+    */
```

Comment:
With the work in #603, this particular piece of code has moved out of here - and the code in #603 will need updating to take into account this logic.

Rather than add an inline todo comment, this should be added as a comment to the other PR so it is implemented before that PR is merged.
```suggestion
```

Reply:
NP. I will do.  However, before I do,  Ben had asked me to put that comment in as there was something he needed to do in this area (so was a reminder to him)

@hardillb before I remove this comment, could you let me know if I should also remove the existing (commented out code) below it?

Thanks.

Reply:
@Steve-Mcl the commented out code gets removed in #603 - please don't touch it otherwise we will get unnecessary merge conflicts to sort out.

Reply:
Ah. I should have looked. Np.  Sorry for the noise.